examples/dev_flow/sanitizers/compiler_sanitizers.rst

```diff
@@ -0,0 +1,354 @@
+.. _examples_dev_flow_sanitizers_compiler_sanitizers:
+
+Compiler sanitizers
+===================
+
+.. warning::
+
+   Using sanitizers in production, particularly with SUID binaries, is dangerous. The libsanitizer
+   runtimes rely on environment variables that could enable privilege escalation attacks.
+   Use sanitizers only in development and testing environments.
```

Comment:
I think something like this would read better:

```
Using sanitizers in production, especially with programs that run with elevated privileges (for example, SUID binaries on Linux), is dangerous.
The sanitizer runtime libraries depend on environment variables, which could allow privilege escalation attacks.
Use sanitizers only in development and testing environments.
```